src/components/Circles.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import styles from "./Circles.module.css";
+
+interface CirclesProps {
+  animationStarted: boolean; // Define the className prop
+}
+
+function Circles(props: CirclesProps) {
+
+  /** 
+   * the circles are arranged in such way:
+   *      top
+   * 1  2  3  4  5
+   *      bot
+   */     
+
+  const circleOneClass = [styles.circleOne, styles.bigCircle, props.animationStarted ? styles.moveLeft : ""].join(" ");
+  const circleTwoClass = [styles.circleTwo, styles.bigCircle, props.animationStarted ? styles.moveLeft : ""].join(" ");
```

Comment:
**suggestion**: You can simplify your workflow using css modules using this library: https://github.com/JedWatson/classnames